Unable to get PNG from SHP in bleed

The same Utility commands which works in release, does not work in bleed anymore and crashes.
Example:
```
$ mono --debug /home/ihptru/Downloads/OpenRA/OpenRA.Utility.exe ra --png /home/ihptru/Downloads/mech.shp /home/ihptru/Downloads/temperat.pal
```

```
Error: Utility application crashed. See utility.log for details

Unhandled Exception:
System.IO.FileNotFoundException: File not found: /home/ihptru/Downloads/mech.shp
File name: '/home/ihptru/Downloads/mech.shp'
  at OpenRA.FileSystem.FileSystem.Open (System.String filename) [0x00031] in /home/ihptru/Downloads/OpenRA/OpenRA.Game/FileSystem/FileSystem.cs:239 
  at OpenRA.Graphics.SpriteLoader.GetFrames (IReadOnlyFileSystem fileSystem, System.String filename, OpenRA.Graphics.ISpriteLoader[] loaders) [0x00003] in /home/ihptru/Downloads/OpenRA/OpenRA.Game/Graphics/SpriteLoader.cs:79 
  at OpenRA.Mods.Common.UtilityCommands.ConvertSpriteToPngCommand.Run (OpenRA.ModData modData, System.String[] args) [0x00000] in <filename unknown>:0 
  at OpenRA.Utility.Program.Main (System.String[] args) [0x0015e] in /home/ihptru/Downloads/OpenRA/OpenRA.Utility/Program.cs:84 
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: File not found: /home/ihptru/Downloads/mech.shp
File name: '/home/ihptru/Downloads/mech.shp'
  at OpenRA.FileSystem.FileSystem.Open (System.String filename) [0x00031] in /home/ihptru/Downloads/OpenRA/OpenRA.Game/FileSystem/FileSystem.cs:239 
  at OpenRA.Graphics.SpriteLoader.GetFrames (IReadOnlyFileSystem fileSystem, System.String filename, OpenRA.Graphics.ISpriteLoader[] loaders) [0x00003] in /home/ihptru/Downloads/OpenRA/OpenRA.Game/Graphics/SpriteLoader.cs:79 
  at OpenRA.Mods.Common.UtilityCommands.ConvertSpriteToPngCommand.Run (OpenRA.ModData modData, System.String[] args) [0x00000] in <filename unknown>:0 
  at OpenRA.Utility.Program.Main (System.String[] args) [0x0015e] in /home/ihptru/Downloads/OpenRA/OpenRA.Utility/Program.cs:84
```

File exists on given path.

--- a/OpenRA.Game/Graphics/SpriteLoader.cs
+++ b/OpenRA.Game/Graphics/SpriteLoader.cs
@@ -78,13 +78,22 @@ namespace OpenRA.Graphics
 		{
 			using (var stream = fileSystem.Open(filename))
 			{
-				ISpriteFrame[] frames;
-				foreach (var loader in loaders)
-					if (loader.TryParseSprite(stream, out frames))
-						return frames;
+				var spriteFrames = GetFrames(stream, loaders);
+				if (spriteFrames == null)
+					throw new InvalidDataException(filename + " is not a valid sprite file!");
 
-				throw new InvalidDataException(filename + " is not a valid sprite file!");
+				return spriteFrames;
 			}
+		}
+
+		public static ISpriteFrame[] GetFrames(Stream stream, ISpriteLoader[] loaders)
+		{
+			ISpriteFrame[] frames;
+			foreach (var loader in loaders)
+				if (loader.TryParseSprite(stream, out frames))
+					return frames;
+
+			return null;
 		}
 	}
 }

--- a/OpenRA.Mods.Common/UtilityCommands/ConvertSpriteToPngCommand.cs
+++ b/OpenRA.Mods.Common/UtilityCommands/ConvertSpriteToPngCommand.cs
@@ -47,7 +47,7 @@ namespace OpenRA.Mods.Common.UtilityCommands
 
 			var palette = new ImmutablePalette(args[2], shadowIndex);
 
-			var frames = SpriteLoader.GetFrames(modData.DefaultFileSystem, src, modData.SpriteLoaders);
+			var frames = SpriteLoader.GetFrames(File.OpenRead(src), modData.SpriteLoaders);
 
 			var usePadding = !args.Contains("--nopadding");
 			var count = 0;

